src/main/kotlin/pl/edu/uj/ii/ksi/mordor/services/thumbnail/ThumbnailChainOfResponsibility.kt

```diff
@@ -0,0 +1,25 @@
+package pl.edu.uj.ii.ksi.mordor.services.thumbnail
+
+import java.io.File
+import org.apache.tika.Tika
+import org.springframework.stereotype.Service
+
+@Service
+class ThumbnailChainOfResponsibility(private val tika: Tika) : ThumbnailExtractor() {
```

Comment:
Rename to ThumbnailAutoExtractor or something else, that actually explains that it does.